src/generate.js

```diff
@@ -55,6 +193,27 @@ const runStringHandlers = (declarations, stringHandlers) => {
     return result;
 };
 
+/**
+ * Generate a CSS ruleset with the selector and containing the declarations.
+ *
+ * This function assumes that the given declarations don't contain any special
+ * children (such as media queries, pseudo-selectors, or descendant styles).
+ *
+ * Example:
+ * ```
+ * generateCSSRuleset(".blah", { color: "red" });
```

Comment:
Would be good to include an example that involves a pseudo selector so show that `selector` isn't necessarily just a classname selector